Rebus/CloudEventify.Rebus.Tests/SerializerTests.cs

```diff
@@ -15,6 +16,32 @@ namespace CloudEventify.Rebus.Tests;
 
 public class SerializerTests
 {
+    [Fact]
+    public async Task Send_With_Source_override()
+    {
+        var hypothesis = Hypothesis.For<A.UserLoggedIn>()
+            .Any(x => x.Id == 1234);
```

Comment:
you could probably check the `MessageContext` here to make it even a better test